simplivity/resources/omnistack_clusters.py

```diff
@@ -116,3 +116,19 @@ def __init__(self, connection, resource_client, data):
         self.data = data
         self._connection = connection
         self._client = resource_client
+
+    def get_connected_clusters(self):
+        """Retrieves directly connected omnistack_clusters.
+
+        Returns:
+            list: List of omnistack_clusters objects.
+        """
+        method_url = "{}/{}/connected_clusters".format(URL, self.data["id"])
+        connected_clusters = self._client.do_get(method_url).get("omnistack_clusters", [])
+
+        cluster_objs = []
+        for cluster in connected_clusters:
+            obj = OmnistackClusters(self._connection).get_by_id(cluster["id"])
```

Comment:
add `self._clusters = OmnistackClusters(self._connection)` to def __init__

Reply:
Updated as per suggestion.

Reply:
@KalpanaBhardwaj, the suggestion was to add the OmsistackClusters object declaration to the constructor so that it can be used in other methods also.

Reply:
Moved the self._clusters = OmnistackClusters(self._connection) in the constructor